test/icu.macro.spec.js

```diff
@@ -19,81 +19,81 @@ pluginTester({
   babelOptions: { filename: __filename, parserOpts: { plugins: ['jsx'] } },
   tests: [
     `
-      import { Trans } from '../icu.macro'
+      import { Trans } from '../../../icu.macro'
```

Comment:
babel-plugin-tester v11 has na updated structure so the import parts had to be updated as well.